modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -6,12 +6,7 @@
 [id="configuration-for-sending-metrics-to-the-monitoring-stack_{context}"]
 = Configuration for sending metrics to the monitoring stack
 
-One of two following custom resources (CR) configures the sending of metrics to the monitoring stack:
-
-* OpenTelemetry Collector CR
-* Prometheus `PodMonitor` CR
-
-A configured OpenTelemetry Collector CR can create a Prometheus `ServiceMonitor` CR for scraping the Collector's pipeline metrics and the enabled Prometheus exporters.
+You can configure the `OpenTelemetryCollector` custom resource (CR) to create a Prometheus `ServiceMonitor` CR or a `PodMonitor` CR for a sidecar deployment. A `ServiceMonitor` can scrape Collector's internal metrics endpoint and Prometheus exporter metrics endpoints.
```

Comment:
```suggestion
You can configure the `OpenTelemetryCollector` custom resource (CR) to create a Prometheus `ServiceMonitor` CR or a `PodMonitor` CR for a sidecar deployment. A `ServiceMonitor` can scrape the Collector's internal metrics endpoint and Prometheus exporter metrics endpoints.
```

Reply:
I think there is a missing article here for that noun. It reads awkward without it. Everywhere else in the docs it is `the Collector's`.

Reply:
It is missing indeed. Thank you for spotting it.